packages/ui/src/ui/pages/tablet_cell_bundles/bundles/BundleEditorDialog/BundleEditorDialog.tsx

```diff
@@ -654,25 +637,58 @@ export function BundleEditorDialog() {
             onAdd={onSubmit}
             onClose={onClose}
             fields={fields}
+            validate={validateFormValues.bind(null, orchidData?.resource_quota)}
         />
     );
 }
 
-function validateMemoryLimits(values: BundleEditorDialogFormValues) {
-    if (values.resources?.tablet_node_resource_guarantee) {
-        const memoryLimits = values.resources.tablet_node_resource_guarantee?.memory || 0;
-        const currentAccLimit =
-            Object.values(values.memory_limits || {}).reduce((acc: number, v: number | boolean) => {
-                if (typeof v !== 'number') {
-                    return acc;
-                }
-                return acc + v;
-            }, 0) || 0;
-
-        if (currentAccLimit > memoryLimits) {
-            return 'The sum of the memory limits exceeds the allowed values';
-        }
+function validateFormValues(
+    quota: OrchidBundleResource | undefined,
+    values: BundleEditorDialogFormValues,
+) {
+    const res: Pick2<typeof values, 'resources', 'info', string> &
+        Pick2<typeof values, 'memory_limits', 'memory_reset', string> = {
+        resources: {info: ''},
+        memory_limits: {memory_reset: ''},
+    };
+
+    const {resources} = values;
+
+    const currentAccLimit =
+        Object.values(values.memory_limits || {}).reduce((acc: number, v: number | boolean) => {
```

Comment:
Reduce always returns its value. The construction with || 0 is not needed
I suggest we redo it for this:
`const currentAccLimit = Object.values(values.memory_limits || {})
  .filter((v: number | boolean): v is number => typeof v === 'number')
  .reduce((acc: number, v: number) => acc + v, 0);`

Reply:
Additional filtering before `reduce`-call is not the best approach cause it brings 2*N instead of N iterations in worst case, also it brings unnecessary allocation of a new array with intermediate elements.

Reply:
Unnecessary `|| 0` is removed